docs/conga-map-backup-workaround.md

```diff
@@ -67,3 +67,90 @@ Remember to point to correct backup folder because we were putting in the name t
 ```bash
 ssh root@congaip "rm -rf /mnt/UDISK/log && cp -R /mnt/UDISK/log.backup /mnt/UDISK/log"
 ```
+
+## Automating the map backup
+
+If your conga looses its map every few days, a workaround could be to automate restoring the map every day using cron.
+
+### Instructions
+
+#### 1. Write a script like this in `/mnt/UDISK/logmaps.sh`:
```

Comment:
The script is located here: https://github.com/freeconga/stuff/blob/master/other/logmaps.sh

Please reference to the file instead of writing it in the README